### ISSUE TITLE
ST0006 As a TL I would to check the report consistency, identify and flag inconsistencies so that we can improve confidence in the report.

### ISSUE THREAD

**Issue by TWJW-SANGER** [sanger/General-Backlog-Items#394]

**Comment by sabrine33** [comment]
Better to look into this after #404  is resolved, as that should update/populate some of the event dates

**Comment by khelwood** [comment]
* Total rows: 34117
* `labware_received` is null but other dates are present: 4765
* `library_start_first` is null but `sequencing_run_start_first` or `sequencing_qc_complete_first` is not null: 7049
* `sequencing_run_start_first` is null but `sequencing_qc_complete_first` is not null: 426

**Comment by khelwood** [comment]
* Look at what studies exhibit these problems.
* Look at what barcode prefixes.
* Look at the age of the labware exhibiting these problems (when was the last time it happened?)

**Comment by khelwood** [comment]
# Total rows: 34117


## labware_received is null but other dates are present: 4765

Study name
 * 1098-PF-ET-GOLASSA : 1
 * 1236-VO-TZ-OKUMU : 6
 * 1282-PF-SS-MANGANO-SM : 1
 * 1295-PF-DO-CRUZ-SM : 1
 * 1334-PF-GH-AMENGA-ETEGO-LAB : 2
 * 1337-PF-GH-ANINAGYEI : 1
 * 1350-PF-MZ-MAYOR : 1
 * 1354-VO-KE-DONNELLY : 6
 * 959 Nematode Genomes : 33
 * ANOSPP : 242
 * Analysis of mutations in mouse spermatogenesis_cDNA_PacBio : 1
 * BAdASS Project (Best Anopheles Assembly Project) : 9
 * BGE_ERGA_DNA : 3
 * BIOSCAN UK for flying insects : 68
 * Blaxter_Single nematode genomes : 8
 * Blood single cell colony mutational and clonal landscape : 4
 * Brain Disease Wellcome Leap Delta Tissue RNA : 1
 * CTVT PacBio Whole Genome Sequencing : 1
 * Cardinal ELGH : 72
 * Cardinal UKB : 56
 * Chip-seq oesophageal adenocarcinoma : 1
 * Clonal selection after gene therapy in SCD - Duplex sequencing : 1
 * Combination Immune Checkpoint Inhibition in Australian Rare Cancers_WES : 2
 * Comparative genomics of Schistosoma : 1
 * DTOL_Darwin R&D : 221
 * DTOL_Darwin Tree of Life : 3287
 * Dummy EGA accessioning study : 31
 * Evolution of MSI status in cancer : 1
 * Flying Insect Long Read : 5
 * G3031_Transposon insertional mutagenesis of the Plasmodium falciparum genome : 12
 * G3169_PWGS_Juno: global genomic survey of Streptococcus agalactiae : 1
 * GAinS_genotyping : 13
 * GOSH Childhood Tumours Diseased Behjati CRUK DNA : 1
 * Genomic analysis of the Baby Biome Study culture collection : 5
 * Genomic assessment of NDM-1 plasmid transmission : 1
 * GridION_testing : 1
 * HCA Blood Skin Disease Atopic dermatitis Wellcome Managed Access - RNA : 1
 * HCA Bone Marrow Wellcome LEAP RNA : 1
 * HCA Heart Adult BHF-DZHK Spatial : 1
 * HG_BL_Barcode association for PDZ combinatorial library : 1
 * HG_CD-METARESPONSE Metagenomics : 1
 * HG_COVID_monocytes : 1
 * HG_IBD-RESPONSE Metagenomics (PREDICCT) : 1
 * HG_Medaka Sample Collection Karlsruhe Institute of Technology : 100
 * HG_WC11263_Systematic assessment of allostery in a model GPCR : 1
 * HG_b2AR-barcode satmut pacbio : 2
 * HMW DNA extraction & Long-read WGS optimization : 17
 * Heron Project R & D : 4
 * I3551_PWGS_Euro-GASP The European Gonococcal Antimicrobial Surveillance Programme (2018 survey) : 1
 * ILB Global Pneumococcal Sequencing (GPS) study I (JP) : 5
 * Investigating early host intestinal epithelial cells - whipworm interactions : 1
 * Investigating the clonal dynamics in interstitial lung diseases using laser-captured microdissection : 2
 * Kings Hepatoblastoma Behjati RNA Managed Access : 1
 * LCMB_WGS_masld_NovaSeqX : 1
 * Long Read DNA testing : 5
 * Lymphocyte colony WGS : 3
 * MRSA_sequencing_as_infection_control : 9
 * Mouse Genome Sequencing : 2
 * Mouse Gut Wellcome Spatial : 1
 * MutWP1: CRUK Grand Challenge Mutographs of Cancer: Colorectal - NCCJ : 1
 * MutWP1: CRUK Grand Challenge Mutographs of Cancer: kidney_CAGEKID : 2
 * NeuroID scRNAseq : 1
 * Oesophageal Adenocarcinoma Organoid ATAC : 1
 * Oesophageal Adenocarcinoma Organoid ONT : 4
 * Oesophageal Adenocarcinoma Organoid PacBio : 2
 * Oesophageal Adenocarcinoma Organoids Iso-Seq : 2
 * Orphan Tumour Study_NB_sNuc : 1
 * PTA (thyroid) : 2
 * Population-based analysis of POT1 variants in a cutaneous melanoma case-control cohort : 65
 * RVI Bait Capture R&D : 2
 * Reference Standards (PCR-free) : 3
 * Saturation Genome Editing of BAP1 : 4
 * SciOps PacBio controls : 3
 * Single molecule somatic mutation detection : 21
 * Sperm PacBio CCS sequencing : 1
 * Springtail sequencing : 2
 * Streptococcus pneumoniae R6x cps99-4038 genome sequencing and assembly : 1
 * Synthetic Genomics : 32
 * TOL_Durb_TFPM-430- Malawi cichlids_DNA : 10
 * Targeted NanoSeq(Airway) -- exploring the influence of mutagens on the genome in single-cell derived lung colonies : 1
 * Team 112 R&D : 4
 * Team51_Development : 5
 * Telomere-2-Telomere sequencing of 12 Species from the Tree of Life : 2
 * Test ENA study : 6
 * The saliva microbiome using nanopore sequencing : 1
 * ToL Meiofauna R&D : 1
 * ToL_Blaxter_ Reference Genomes_ DNA : 52
 * ToL_Blaxter_ WC10686 _ ERGA PI_ DNA : 2
 * ToL_Blaxter_Ormia ochracea_LRES WGS : 9
 * ToL_Durbin_G2756_DNA : 1
 * ToL_Durbin_Yponomeuta ermine moth popgen 2022 : 2
 * ToL_Jaron_ReferenceGenomes_DNA : 1
 * ToL_Meier_ReferenceGenomes : 66
 * ToL_Reference Genomes_DNA : 1
 * Tree of Life - ASG : 156
 * Tree of Life - VGP : 8
 * Typhimurium_Africa : 1
 * UCL_Mice_Lung (WG)(Novaseq) : 1
 * Using genetics to identify cell types and mechanisms underlying susceptibility to primary sclerosing cholangitis - RNA : 12
 * Using genetics to identify cell types and mechanisms underlying susceptibility to primary sclerosing cholangitis [omni2.5-8ex_prscch](prscch) : 1
 * Vento_Placental Cell Atlas : 1
 * Vertebrate Genome Project_FISH : 1
 * WC10290_Antimicrobial resistance of Escherichia coli isolates from stool, invasive disease and environmental swabs among vulnerable populations in Kenya : 1
 * WC10811_PWGS_Pilot study to scale-up DNA extractions for metagenomics : 1
 * WC11016_Pseudomonas aeruginosa deep sequencing_ISC : 2
 * WC11063_Sputum Metagen : 1
 * WGS_Somatic mutation in Primary Sjogren’s Syndrome : 1
 * WGS_non-malignant-colon_and_colorectal-cancer : 1

Latest date in row
 * 2022-06 : 238
 * 2022-07 : 143
 * 2022-08 : 119
 * 2022-09 : 337
 * 2022-10 : 269
 * 2022-11 : 161
 * 2022-12 : 256
 * 2023-01 : 252
 * 2023-02 : 238
 * 2023-03 : 260
 * 2023-04 : 195
 * 2023-05 : 183
 * 2023-06 : 231
 * 2023-07 : 128
 * 2023-08 : 99
 * 2023-09 : 116
 * 2023-10 : 144
 * 2023-11 : 95
 * 2023-12 : 117
 * 2024-01 : 55
 * 2024-02 : 62
 * 2024-03 : 220
 * 2024-04 : 202
 * 2024-05 : 422
 * 2024-06 : 223

Barcode prefix
 * [none] : 55
 * CGAP- : 3
 * DN : 40
 * FB : 53
 * FD : 1137
 * FF : 616
 * FR : 120
 * FS : 1650
 * NT : 339
 * S : 72
 * SQPP- : 680


## library_start_first is null but sequencing_run_start_first or sequencing_qc_complete_first is not null: 7049

Study name
 * 1208-PF-LA-CMPE-GENRE : 4
 * 1236-VO-TZ-OKUMU : 1
 * 1270-VO-MULTI-PAMGEN : 1
 * 1315-VO-NG-OMITOLA : 1
 * 2021 WGS cichlids from lab crosses : 1
 * 959 Nematode Genomes : 2
 * Brain Disease Wellcome Leap Delta Tissue RNA : 1
 * Breast CIS_WGS : 4
 * Bulk sequencing study for human male germline : 1
 * COVID-19 Challenge 2 WOOE RNA Managed Access : 1
 * Cardinal ELGH : 1223
 * Cardinal UKB : 5043
 * Clonal dynamics and mutation burden in male germline - DupSeq : 1
 * DTOL_Darwin Tree of Life : 124
 * Dual RNA-seq of TPA - pilot : 1
 * Endometriosis : 1
 * G3031_Transposon insertional mutagenesis of the Plasmodium falciparum genome : 2
 * Genomic landscape of pre-malignant and malignant lung tissue : 2
 * GermCellTumour : 1
 * HCA Blood Skin Disease Atopic dermatitis Wellcome Managed Access - RNA : 1
 * HCA Bone Marrow Wellcome LEAP RNA : 1
 * HCA Cartilage hIPSCs hESCs Sid Lawrence Wellcome Fellowship RNA Managed Access : 1
 * HCA Female Reproductive Paediatric CZI RNA : 1
 * HCA Heart Adult BHF-DZHK Spatial : 1
 * HCA Hormone glands Vento RNA : 1
 * HCA Immune Adult Vento RNA Managed Access : 1
 * HCA Immune Aging Biology RNA : 1
 * HCA Skin Foetal Haniffa RNA : 1
 * HCA Thymus Disease : 1
 * HCA_Oesophagus_Regeneration_CASM_RNA : 1
 * HG_ Mapping genetic variants underlying gene regulation in healthy intestinal cell types to identify novel IBD drug targets : 2
 * HG_CCF_ Mapping genetic variants underlying gene regulation in inflammed intestinal cell types to identify novel IBD drug targets : 2
 * HG_GT_ED_OTAR2064 - SLE Map Main Study: Single Cell Profiling of SLE : 5
 * HG_IBD-RESPONSE : 1
 * HG_Identifying the effects of human gene knockouts in immune cells using single-cell transcriptomics : 1
 * HG_WC10130_IBD Bioresource Whole Exome Sequencing of 20,000 samples : 1
 * HG_WC10289_Perturbing T cell activation circuitries at single cell resolution : 1
 * HG_scRNA-seq CD blood : 2
 * Healthy ageing thymus : 1
 * Investigating early host intestinal epithelial cells - whipworm interactions : 1
 * MSI Cancer Models - WGS : 1
 * Monozygotic twins with hereditary spherocytosis and MPNHereditary spherocytosis in twins : 2
 * MutWPX: CRUK Grand Challenge Mutographs of Cancer: Effects of Chemotherapy on the Somatic Mutational Landscape in Normal Human Tissue - DuplexSeq : 2
 * Mutational variability in oesophagus in areas of high SCC incidence : 1
 * Neuroinflammation DNAseq : 1
 * Orphan Tumour Study_NB_sNuc : 2
 * Paediatric_CNS_tumour_autopsy_WGMS : 1
 * Panbody_nanoseq : 2
 * Reference Standards (PCR-free) : 1
 * Reference Standards project : 1
 * Single Cell Atlases : 1
 * Single cell multiomics study of male germline-RNA : 2
 * Targeted NanoSeq_Bladder : 1
 * Targeted NanoSeq_Pan-body : 1
 * Targeted NanoSeq_Pancreas : 1
 * Targeted NanoSeq_Prostate : 1
 * Targeted NanoSeq_Sperm : 1
 * Targeted NanoSeq_Synovium : 1
 * The Little Princess Knowledge Bank : 2
 * The Little Princess Knowledge Bank_RNAseq : 1
 * The evolution of CML - WGS : 7
 * Transcriptional Consequences of Copy Number Changes_MY_HDBR_200531 : 1
 * Tree of Life - ASG : 7
 * Understanding hematopoietic stem cell mobilization and engraftment : 1
 * Vaccine Pertubation Study pilot experiments : 559
 * WC10811_PWGS_Pilot study to scale-up DNA extractions for metagenomics : 1
 * WGS_11pcw_fetus_hdbr_15951_DNA : 3
 * scRNAseq of transmissible cancers_10X : 1

Latest date in row
 * 2022-06 : 875
 * 2022-07 : 343
 * 2022-08 : 1071
 * 2022-09 : 706
 * 2022-10 : 601
 * 2022-11 : 511
 * 2022-12 : 607
 * 2023-01 : 711
 * 2023-02 : 606
 * 2023-03 : 460
 * 2023-04 : 307
 * 2023-05 : 176
 * 2023-06 : 2
 * 2023-07 : 6
 * 2023-08 : 7
 * 2023-09 : 3
 * 2023-10 : 13
 * 2023-11 : 4
 * 2024-01 : 1
 * 2024-02 : 1
 * 2024-03 : 7
 * 2024-04 : 21
 * 2024-05 : 10

Barcode prefix
 * [none] : 4996
 * DN : 66
 * FB : 1
 * FD : 18
 * FF : 103
 * FR : 6
 * FS : 3
 * NT : 629
 * S : 1202
 * SQPP- : 25


## sequencing_run_start_first is null but sequencing_qc_complete_first is not null: 426

Study name
 * 1270-VO-MULTI-PAMGEN : 1
 * DTOL_Darwin Tree of Life : 45
 * MSI Cancer Models - WGS : 1
 * MutWPX: CRUK Grand Challenge Mutographs of Cancer: Effects of Chemotherapy on the Somatic Mutational Landscape in Normal Human Tissue - DuplexSeq : 1
 * Neuroinflammation DNAseq : 1
 * The Little Princess Knowledge Bank : 1
 * Transcriptional Consequences of Copy Number Changes_MY_HDBR_200531 : 1
 * Tree of Life - ASG : 1
 * Vaccine Pertubation Study pilot experiments : 374

Latest date in row
 * 2022-06 : 413
 * 2022-10 : 3
 * 2022-12 : 2
 * 2023-08 : 1
 * 2024-03 : 3
 * 2024-04 : 2
 * 2024-05 : 2

Barcode prefix
 * DN : 6
 * FD : 13
 * FF : 29
 * FR : 2
 * FS : 2
 * NT : 374

**Comment by TWJW-SANGER** [comment]
Adding some notes of my own.

RE: labware_received is null but other dates are present
Looking at the first study that has this problem 1098-PF-ET-GOLASSA gives 2 plates SQPP-52926-B and SQPP-52927-C. Only SQPP-52927-C has NULL for labware_received.

Using the query on confluence to Pull All Event Data For A Piece of Labware for SQPP-52927-C gives only events of type  sample_manifest.created, labwhere_create and sample_manifest.updated.

Looking at the MLWH_Events 101 page for 'labware_received'  it says _When a labware is received via Sequencescape Labwhere Reception_.  Creating this event in SequenceScape allows it to have a sample subject as SequenceScape knows about samples.

The labwhere_create event_type is created by LabWhere (which doesn't know about samples).

What I think is happening is that sometimes the lab are scanning in labware into SequenceScape's Labwhere reception page and sometimes the lab are scanning in labware straight into the LabWhere application.

We probably need another story to look at finding the first scanned in event between labwhere_create and labware_received events. NB This is complicated by the first CTE in the sample_tracking_vew which is filtering on the sample role that the labwhere_create event will not have....

**Comment by khelwood** [comment]
@tw6 It's looks to me like the reason for the _sequencing_run_start_first is null but sequencing_qc_complete_first is not null_ rows is where the `sequencing_start` event is before the date cutoff for events (two years ago). If that's a problem, we'd have to include older events and join them onto the query.

**Comment by TWJW-SANGER** [comment]
Ok,  an artifact of the cutoff date for that one. I think that is acceptable and we can add a note that this can happen in the view.

**Comment by TWJW-SANGER** [comment]
Check IRods locations when no sequencing run dates have been created.

**Comment by khelwood** [comment]
## sequencing_run_start_first is null but irods_root_collections is not null: 8975

Study name
 * 1146-PF-MULTI-PRICE : 1
 * 1162-PF-GM-NGWA : 5
 * 1197-PF-ML-DIAKITE-SM : 7
 * 1200-PF-GH-MAIGA-SM : 4
 * 1202-PF-MY-ANSTEY : 5
 * 1208-PF-LA-CMPE-GENRE : 21
 * 1209-PF-VN-IMPEQN-GENRE : 28
 * 1219-PV-LA-CMPE-GENRE : 18
 * 1227-PF-PE-VALDIVIA : 3
 * 1230-VO-MULTI-AYALA : 5
 * 1236-VO-TZ-OKUMU : 4
 * 1241-PF-GH-ANINAGYEI : 7
 * 1250-PV-PE-VALDIVIA : 6
 * 1270-VO-MULTI-PAMGEN : 7
 * 1280-VO-ZA-MUNHENGA : 4
 * 1281-VO-CM-CHRISTOPHE : 3
 * 1283-PF-SD-MOHAMED-SM : 9
 * 1288-VO-UG-DONNELLY : 6
 * 1315-VO-NG-OMITOLA : 2
 * 2021 WGS cichlids from lab crosses : 1
 * 959 Nematode Genomes : 24
 * ANOSPP : 78
 * ARCHIVED - HCA Brain Paediatric Bayraktar RNA Managed Access : 1
 * Analysis of drug resistant parasites generated by drug-pressure or CRISPR/Cas9 editing. : 6
 * BAdASS Project (Best Anopheles Assembly Project) : 24
 * Bottleneck Sequencing of Animal Tissue [WGS] : 2
 * Bottleneck Sequencing of Human Tissue - (WGS) : 9
 * Brain Disease Wellcome Leap Delta Tissue RNA : 1
 * Breast CIS_WGS : 6
 * CGP CORE CELL LINES : 726
 * COVID-19 Challenge COVHIC Wellcome RNA Managed Access : 6
 * CRUK-Pilot-ORG : 51
 * Cancer Cell Model RNA Sequencing. : 1
 * Clonal dynamics and mutation burden in male germline - DupSeq : 4
 * Comparative genomics of Schistosoma : 7
 * Comparison of PNG and Gabon hookworms : 2
 * DERMATLAS: Hidradenoma_RNAseq : 2
 * DERMATLAS: Leiomyoma_RNAseq : 1
 * DERMATLAS: Leiomyosarcoma_RNAseq : 2
 * DERMATLAS: Poroma_RNAseq : 1
 * DERMATLAS: SG basal cell adenoma and adenocarcinoma_RNAseq : 1
 * DTOL_Darwin R&D : 11
 * DTOL_Darwin Tree of Life : 639
 * DTOL_Darwin Tree of Life RNA : 7
 * DTOL_Darwin Vertebrate Genomes : 3
 * Endometriosis : 7
 * Epigenetic regulation of pre-implantation development : 130
 * Evolution of MSI status in cancer : 1
 * Exploring the influence of mutagens on the genome in single-cell derived lung colonies_Duplex : 1
 * Exploring the influence of mutagens on the transcriptome in single-cell derived lung colonies_RNA : 1
 * Exploring the landscape of somatic mutations in human alveolar cells : 2
 * G2126_PWGS_Phenomic characterization of Psoriatic Arthritis: defining the role of the microbiome : 6
 * G2614_Gut dysbiosis in cystic fibrosis : 3
 * G2803_ISC_Stronger-SAFE: Understanding transmission and intervention trials for an enhanced safe strategy for Trachoma control : 3
 * G2851_ISC_Pilot_Understanding differences in clinical presentations of leishmaniasis in an endemic disease focus : 1
 * G2851_WGS_Understanding differences in clinical presentations of leishmaniasis in an endemic disease focus : 2
 * G3031_Transposon insertional mutagenesis of the Plasmodium falciparum genome : 2
 * G3169_PWGS_Juno: global genomic survey of Streptococcus agalactiae : 67
 * G3227_PWGS_Maximizing the knowledge gained from California Senate Bill 27 : 79
 * G3360_PWGS_Understanding the emergence of epidemic Vibrio cholerae from its global source : 35
 * G3377_WGS_Opisthorchis viverrini population genomics : 1
 * G3399_PWGS_Dissecting_the_genetic_of_melioidosis_infection_II : 3
 * G3399_RNA_Melioidosis host transcriptome : 8
 * G3428_PWGS_Inferring pneumococcal transmission routes to infants during a cluster randomised vaccine trial in Nha Trang, Vietnam : 6
 * Genetic Vulnerabilities in Cancer : 1
 * Genomic analysis of the Baby Biome Study culture collection : 13
 * Genomic landscape of pre-malignant and malignant lung tissue : 8
 * Genomics of acral lentiginous melanoma : 5
 * Genomics of acral lentiginous melanoma - RNAseq : 3
 * GermCellTumour : 20
 * HCA Adult Immune ThDefine RNA : 3
 * HCA Bone Foetal WSSS Dev RNA : 1
 * HCA Bone Marrow Wellcome LEAP RNA : 3
 * HCA Bone Tumour RNOH RNA : 5
 * HCA Brain Disease Downs Bayraktar RNA : 3
 * HCA Female Reproductive Paediatric CZI RNA : 2
 * HCA Fetal gonads HUGODECA : 8
 * HCA Heart Adult BHF-DZHK Spatial : 2
 * HCA Heart Foetal BHF-DZHK RNA : 2
 * HCA Hormone glands Vento RNA : 3
 * HCA Immune Adult CZI Pan Immune Aging RNA : 2
 * HCA Immune Disease TCL Wellcome RNA Managed Access : 2
 * HCA Kidney Adult CZI RNA : 1
 * HCA Lung Adult discovAIR DNA : 1
 * HCA Lung Adult discovAIR RNA : 3
 * HCA Lung Foetal WSSS RNA : 1
 * HCA Thymus Disease : 4
 * HCA_Cambridge_sCells : 52
 * HG_ Mapping genetic variants underlying gene regulation in healthy intestinal cell types to identify novel IBD drug targets : 43
 * HG_CCF_ Mapping genetic variants underlying gene regulation in inflammed intestinal cell types to identify novel IBD drug targets : 33
 * HG_Development Disorders in Africa : 7
 * HG_Effect of ADCY7 risk variant on immune cells : 1
 * HG_IBD Sample collection and QC 2019 : 60
 * HG_RNASeq of cellular DDD models pilot : 474
 * HG_WC10081_IBD 2020 Collection & QC : 4
 * HG_WC10130_IBD Bioresource Whole Exome Sequencing of 20,000 samples : 79
 * HG_WC10162_Birth Cohort_MCS : 88
 * HG_WC10333_ scRNAseq of newly diagnosed IBD : 1
 * HG_scRNA-seq CD blood : 12
 * HSC colony many years post-allogeneic bone marrow transplant - Bait set 2 : 1
 * HSC colony many years post-allogeneic bone marrow transplant_TGS : 2
 * Healthy ageing thymus : 7
 * Heron Project R & D : 9
 * Human Cellular Genetics of Innate Immunity : 5
 * I3498_PWGS_The Human-Animal-Environmental interfaces in infectious diarrhea and the spread of antibiotic resistance : 10
 * I3551_PWGS_Euro-GASP The European Gonococcal Antimicrobial Surveillance Programme (2018 survey) : 29
 * IBD Bioresource [Collection Study] : 28
 * IHTP_PWGS 1132-PF_K1000g_DBS_KE_BEJON : 16
 * IHTP_PWGS 1136-PF-GM-NGWA : 1
 * ILB Global Pneumococcal Sequencing (GPS) study I (JP) : 143
 * ILB Identifying pneumococcal genetic determinants for progression from colonisation to serious disease (JP) : 9
 * ImmunoAgeing_Colonies_WGS : 6
 * Interrogating gastrointestinal tumour resistance and metastasis with functional precision oncology at single-cell resolution : 1
 * Investigating early host intestinal epithelial cells - whipworm interactions : 1473
 * Kidney Single Cell Study : 287
 * Li-Fraumeni Syndrome Inheritance and Tumourigenesis : 4
 * Lymphocyte colony WGS : 18
 * MQ R and D 2014 : 21
 * MSI Cancer Models - WGS : 2
 * Mouse Genome Sequencing : 5
 * MutWP4: CRUK Grand Challenge Mutographs of Cancer: Mutagen-treated organoids : 5
 * MutWP4: CRUK Grand Challenge Mutographs of Cancer: organoid control WGS : 1
 * MutWPX: CRUK Grand Challenge Mutographs of Cancer: Effects of Chemotherapy on the Somatic Mutational Landscape in Normal Human Tissue - DuplexSeq : 9
 * Mutational burden in oesophagus following chemotherapy and radiotherapy treatment : 1
 * Nanoseq of the Ageing Brain [WGS] : 6
 * Nanoseq of the Degenerative Joint Disease [WGS] : 1
 * Neuroinflammation DNAseq : 3
 * Normal and cancer-predisposed kidney : 3
 * Oesophageal Adenocarcinoma Organoid 10x : 1
 * Organoid Derivation Project - GRCh38 - RNAseq : 3
 * Organoid Derivation Project - GRCh38 - WGS : 2
 * Organoid Derivation Project: WGS : 15
 * Orphan Tumour Study_NB_sNuc : 25
 * PID pilot Omenn : 48
 * PWGS_Characterizing the gut microbiota in Latin American populations (Latinbiota): Argentina I : 7
 * PWGS_G2121_Cystic_fibrosis_pathogens : 5
 * PWGS_Latinbiota : 1
 * Pan_body_mouse_WG_Nanoseq : 3
 * Panbody_nanoseq : 3
 * Placental genomics : 6
 * RNA - Genomic profiling of relapse in Adult ALL : 1
 * Reference Standards (PCR-free) : 2
 * Reference Standards project : 1
 * Renal oncocytomas - RNA : 1
 * Rodent RNA sequencing : 1
 * SHK1 epidermis : 8
 * SHK1 tumours : 5
 * Schistosoma mansoni miracidia single cell RNAseq : 3
 * Sequencing of patient samples who received immune checkpoint inhibition - RNAseq - NKI : 5
 * Sequencing of patient samples who received immune checkpoint inhibition - WES - NKI : 4
 * Single Cell Atlases : 32
 * Single cell multiomics study of male germline-RNA : 5
 * Somatic Mutations in Degenerative Disease [exome] : 22
 * Somatic mutation and clonal evolution in the human testes : 15
 * Somatic mutation and clonal evolution in the pancreas_exome : 14
 * Somatic mutations in HSCs of patients with chromosome instability syndromes_WGS : 14
 * Somatic mutations in normal proximal airway epithelium after immunotherapy : 4
 * Somatic mutations in normal proximal airway epithelium after immunotherapy_RNA : 3
 * Spatial transcriptomic profiling of pre-malignant and malignant lung tissue_VISIUM : 1
 * TGS - Gastric Mutational Signatures (GMS) : 12
 * Targeted NanoSeq (TIL) : 1
 * Targeted NanoSeq_Brain and Neurons : 1
 * Targeted NanoSeq_Liver : 1
 * Targeted NanoSeq_Supernatants : 1
 * Team 112 R&D : 11
 * Team51_Development : 277
 * Temporal cell atlas for schistosomes : 1
 * Teratoma_WGS : 7
 * The Little Princess Knowledge Bank : 14
 * The Little Princess Knowledge Bank_RNAseq : 11
 * The effects of chemotherapy and radiation in human oesophageal epithelium- low input : 5
 * The evolution of CML - WGS : 3
 * Thymic epithelial transplantation for complete DiGeorge syndrome : 1
 * Thymic organoid : 4
 * To profile the landscape of sebaceous tumours - RNA : 6
 * ToL Meiofauna R&D : 1
 * Transcriptional Consequences of Copy Number Changes_MY_HDBR_200531 : 14
 * Tree of Life - ASG : 20
 * Tree of Life - VGP : 6
 * UCL_Mice_Lung (WG)(Novaseq) : 2
 * Understanding hematopoietic stem cell mobilization and engraftment : 16
 * Using genetics to identify cell types and mechanisms underlying susceptibility to primary sclerosing cholangitis - RNA : 12
 * VGP_RNASeq : 1
 * Vaccine Pertubation Study pilot experiments : 3068
 * Vento_Placental Cell Atlas : 7
 * Vertebrate Genome Project_FISH : 13
 * WC10290_Antimicrobial resistance of Escherichia coli isolates from stool, invasive disease and environmental swabs among vulnerable populations in Kenya : 6
 * WC10314_RNA_iPS derived intestinal organoids, control untreated or treated inside or outside with CTX 48hr : 4
 * WC10479_PWGS_Group B Streptococcus from maternal carriage in Netherlands : 2
 * WGS_11pcw_fetus_hdbr_15951_DNA : 3
 * WGS_Somatic mutation in Primary Sjogren’s Syndrome : 9

Latest date in row
 * 2022-06 : 3068
 * 2022-07 : 714
 * 2022-08 : 87
 * 2022-09 : 109
 * 2022-10 : 565
 * 2022-11 : 263
 * 2022-12 : 1885
 * 2023-01 : 564
 * 2023-02 : 67
 * 2023-03 : 26
 * 2023-04 : 7
 * 2023-05 : 3
 * 2023-06 : 33
 * 2023-07 : 1
 * 2023-08 : 37
 * 2023-09 : 16
 * 2023-10 : 12
 * 2023-11 : 2
 * 2023-12 : 1
 * 2024-01 : 308
 * 2024-02 : 17
 * 2024-03 : 176
 * 2024-04 : 229
 * 2024-05 : 82
 * 2024-06 : 703

Barcode prefix
 * CGAP- : 192
 * DN : 1688
 * FB : 1
 * FD : 61
 * FF : 199
 * FR : 30
 * FS : 2
 * ND : 9
 * NR : 5
 * NT : 6786
 * PLTE- : 2

**Comment by khelwood** [comment]
## sequencing_qc_complete_first is null but irods_root_collections is not null: 8903

Study name
 * 1146-PF-MULTI-PRICE : 1
 * 1162-PF-GM-NGWA : 5
 * 1197-PF-ML-DIAKITE-SM : 7
 * 1200-PF-GH-MAIGA-SM : 4
 * 1202-PF-MY-ANSTEY : 5
 * 1208-PF-LA-CMPE-GENRE : 21
 * 1209-PF-VN-IMPEQN-GENRE : 28
 * 1219-PV-LA-CMPE-GENRE : 18
 * 1227-PF-PE-VALDIVIA : 3
 * 1230-VO-MULTI-AYALA : 5
 * 1236-VO-TZ-OKUMU : 3
 * 1241-PF-GH-ANINAGYEI : 7
 * 1250-PV-PE-VALDIVIA : 6
 * 1270-VO-MULTI-PAMGEN : 7
 * 1280-VO-ZA-MUNHENGA : 4
 * 1281-VO-CM-CHRISTOPHE : 3
 * 1283-PF-SD-MOHAMED-SM : 9
 * 1288-VO-UG-DONNELLY : 6
 * 1315-VO-NG-OMITOLA : 1
 * 1333-VO-GH-AMENGO-ETEGO : 3
 * 2021 WGS cichlids from lab crosses : 1
 * 959 Nematode Genomes : 24
 * ANOSPP : 78
 * ARCHIVED - HCA Brain Paediatric Bayraktar RNA Managed Access : 1
 * Analysis of drug resistant parasites generated by drug-pressure or CRISPR/Cas9 editing. : 6
 * BAdASS Project (Best Anopheles Assembly Project) : 24
 * Bottleneck Sequencing of Animal Tissue [WGS] : 2
 * Bottleneck Sequencing of Human Tissue - (WGS) : 9
 * Brain Disease Wellcome Leap Delta Tissue RNA : 1
 * Breast CIS_WGS : 6
 * CGP CORE CELL LINES : 726
 * COVID-19 Challenge COVHIC Wellcome RNA Managed Access : 6
 * CRUK-Pilot-ORG : 51
 * Cancer Cell Model RNA Sequencing. : 1
 * Clonal dynamics and mutation burden in male germline - DupSeq : 4
 * Comparative genomics of Schistosoma : 7
 * Comparison of PNG and Gabon hookworms : 2
 * Cross_Species_Targeted_NanoSeq_Germline_Project : 1
 * DERMATLAS: Hidradenoma_RNAseq : 2
 * DERMATLAS: Leiomyoma_RNAseq : 1
 * DERMATLAS: Leiomyosarcoma_RNAseq : 2
 * DERMATLAS: Poroma_RNAseq : 1
 * DERMATLAS: SG basal cell adenoma and adenocarcinoma_RNAseq : 1
 * DTOL_Darwin R&D : 11
 * DTOL_Darwin Tree of Life : 585
 * DTOL_Darwin Tree of Life RNA : 7
 * DTOL_Darwin Vertebrate Genomes : 3
 * Endometriosis : 7
 * Epigenetic regulation of pre-implantation development : 130
 * Evolution of MSI status in cancer : 1
 * Exploring the influence of mutagens on the genome in single-cell derived lung colonies_Duplex : 1
 * Exploring the influence of mutagens on the transcriptome in single-cell derived lung colonies_RNA : 1
 * Exploring the landscape of somatic mutations in human alveolar cells : 2
 * G2126_PWGS_Phenomic characterization of Psoriatic Arthritis: defining the role of the microbiome : 6
 * G2614_Gut dysbiosis in cystic fibrosis : 3
 * G2803_ISC_Stronger-SAFE: Understanding transmission and intervention trials for an enhanced safe strategy for Trachoma control : 3
 * G2851_ISC_Pilot_Understanding differences in clinical presentations of leishmaniasis in an endemic disease focus : 1
 * G2851_WGS_Understanding differences in clinical presentations of leishmaniasis in an endemic disease focus : 2
 * G3031_Transposon insertional mutagenesis of the Plasmodium falciparum genome : 2
 * G3169_PWGS_Juno: global genomic survey of Streptococcus agalactiae : 67
 * G3227_PWGS_Maximizing the knowledge gained from California Senate Bill 27 : 79
 * G3360_PWGS_Understanding the emergence of epidemic Vibrio cholerae from its global source : 35
 * G3377_WGS_Opisthorchis viverrini population genomics : 1
 * G3399_PWGS_Dissecting_the_genetic_of_melioidosis_infection_II : 3
 * G3399_RNA_Melioidosis host transcriptome : 8
 * G3428_PWGS_Inferring pneumococcal transmission routes to infants during a cluster randomised vaccine trial in Nha Trang, Vietnam : 6
 * Genetic Vulnerabilities in Cancer : 1
 * Genomic analysis of the Baby Biome Study culture collection : 13
 * Genomic landscape of pre-malignant and malignant lung tissue : 8
 * Genomics of acral lentiginous melanoma : 5
 * Genomics of acral lentiginous melanoma - RNAseq : 3
 * GermCellTumour : 20
 * HCA Adult Immune ThDefine RNA : 3
 * HCA Bone Foetal WSSS Dev RNA : 1
 * HCA Bone Marrow Wellcome LEAP RNA : 3
 * HCA Bone Tumour RNOH RNA : 5
 * HCA Brain Disease Downs Bayraktar RNA : 3
 * HCA Female Reproductive Paediatric CZI RNA : 2
 * HCA Fetal gonads HUGODECA : 8
 * HCA Heart Adult BHF-DZHK Spatial : 2
 * HCA Heart Foetal BHF-DZHK RNA : 2
 * HCA Hormone glands Vento RNA : 3
 * HCA Immune Adult CZI Pan Immune Aging RNA : 2
 * HCA Immune Disease TCL Wellcome RNA Managed Access : 2
 * HCA Kidney Adult CZI RNA : 1
 * HCA Lung Adult discovAIR DNA : 1
 * HCA Lung Adult discovAIR RNA : 3
 * HCA Lung Foetal WSSS RNA : 1
 * HCA Thymus Disease : 4
 * HCA_Cambridge_sCells : 52
 * HG_ Mapping genetic variants underlying gene regulation in healthy intestinal cell types to identify novel IBD drug targets : 42
 * HG_CCF_ Mapping genetic variants underlying gene regulation in inflammed intestinal cell types to identify novel IBD drug targets : 32
 * HG_Development Disorders in Africa : 7
 * HG_Effect of ADCY7 risk variant on immune cells : 1
 * HG_IBD Sample collection and QC 2019 : 60
 * HG_RNASeq of cellular DDD models pilot : 474
 * HG_WC10081_IBD 2020 Collection & QC : 4
 * HG_WC10130_IBD Bioresource Whole Exome Sequencing of 20,000 samples : 73
 * HG_WC10162_Birth Cohort_MCS : 88
 * HG_WC10333_ scRNAseq of newly diagnosed IBD : 1
 * HG_scRNA-seq CD blood : 11
 * HSC colony many years post-allogeneic bone marrow transplant - Bait set 2 : 1
 * HSC colony many years post-allogeneic bone marrow transplant_TGS : 2
 * Healthy ageing thymus : 7
 * Heron Project R & D : 9
 * Human Cellular Genetics of Innate Immunity : 5
 * I3498_PWGS_The Human-Animal-Environmental interfaces in infectious diarrhea and the spread of antibiotic resistance : 10
 * I3551_PWGS_Euro-GASP The European Gonococcal Antimicrobial Surveillance Programme (2018 survey) : 29
 * IBD Bioresource [Collection Study] : 28
 * IHTP_PWGS 1132-PF_K1000g_DBS_KE_BEJON : 16
 * IHTP_PWGS 1136-PF-GM-NGWA : 1
 * ILB Global Pneumococcal Sequencing (GPS) study I (JP) : 143
 * ILB Identifying pneumococcal genetic determinants for progression from colonisation to serious disease (JP) : 9
 * ImmunoAgeing_Colonies_WGS : 6
 * Interrogating gastrointestinal tumour resistance and metastasis with functional precision oncology at single-cell resolution : 1
 * Investigating early host intestinal epithelial cells - whipworm interactions : 1473
 * Kidney Single Cell Study : 287
 * Li-Fraumeni Syndrome Inheritance and Tumourigenesis : 4
 * Lymphocyte colony WGS : 18
 * MQ R and D 2014 : 21
 * MSI Cancer Models - WGS : 2
 * Mouse Genome Sequencing : 5
 * MutWP4: CRUK Grand Challenge Mutographs of Cancer: Mutagen-treated organoids : 5
 * MutWP4: CRUK Grand Challenge Mutographs of Cancer: organoid control WGS : 1
 * MutWPX: CRUK Grand Challenge Mutographs of Cancer: Effects of Chemotherapy on the Somatic Mutational Landscape in Normal Human Tissue - DuplexSeq : 8
 * Mutational burden in oesophagus following chemotherapy and radiotherapy treatment : 1
 * Nanoseq of the Ageing Brain [WGS] : 6
 * Nanoseq of the Degenerative Joint Disease [WGS] : 1
 * Neuroinflammation DNAseq : 2
 * Normal and cancer-predisposed kidney : 3
 * Oesophageal Adenocarcinoma Organoid 10x : 1
 * Organoid Derivation Project - GRCh38 - RNAseq : 3
 * Organoid Derivation Project - GRCh38 - WGS : 2
 * Organoid Derivation Project: WGS : 15
 * Orphan Tumour Study_NB_sNuc : 25
 * PID pilot Omenn : 48
 * PWGS_Characterizing the gut microbiota in Latin American populations (Latinbiota): Argentina I : 7
 * PWGS_G2121_Cystic_fibrosis_pathogens : 5
 * PWGS_Latinbiota : 1
 * Pan_body_mouse_WG_Nanoseq : 3
 * Panbody_nanoseq : 3
 * Placental genomics : 6
 * RNA - Genomic profiling of relapse in Adult ALL : 1
 * Reference Standards (PCR-free) : 2
 * Reference Standards project : 1
 * Renal oncocytomas - RNA : 1
 * Rodent RNA sequencing : 1
 * SHK1 epidermis : 8
 * SHK1 tumours : 5
 * Schistosoma mansoni miracidia single cell RNAseq : 3
 * Sequencing of patient samples who received immune checkpoint inhibition - RNAseq - NKI : 5
 * Sequencing of patient samples who received immune checkpoint inhibition - WES - NKI : 4
 * Single Cell Atlases : 32
 * Single cell multiomics study of male germline-RNA : 3
 * Somatic Mutations in Degenerative Disease [exome] : 22
 * Somatic mutation and clonal evolution in the human testes : 15
 * Somatic mutation and clonal evolution in the pancreas_exome : 14
 * Somatic mutations in HSCs of patients with chromosome instability syndromes_WGS : 14
 * Somatic mutations in normal proximal airway epithelium after immunotherapy : 4
 * Somatic mutations in normal proximal airway epithelium after immunotherapy_RNA : 3
 * Spatial transcriptomic profiling of pre-malignant and malignant lung tissue_VISIUM : 1
 * Stratton: WGS FFPE studies_HNC : 1
 * TGS - Gastric Mutational Signatures (GMS) : 12
 * Targeted NanoSeq (TIL) : 1
 * Targeted NanoSeq_Brain and Neurons : 1
 * Targeted NanoSeq_Liver : 1
 * Targeted NanoSeq_Supernatants : 1
 * Team 112 R&D : 10
 * Team51_Development : 277
 * Temporal cell atlas for schistosomes : 1
 * Teratoma_WGS : 7
 * The Little Princess Knowledge Bank : 14
 * The Little Princess Knowledge Bank_RNAseq : 11
 * The effects of chemotherapy and radiation in human oesophageal epithelium- low input : 5
 * The evolution of CML - WGS : 1
 * Thymic epithelial transplantation for complete DiGeorge syndrome : 1
 * Thymic organoid : 4
 * To profile the landscape of sebaceous tumours - RNA : 6
 * ToL Meiofauna R&D : 1
 * Transcriptional Consequences of Copy Number Changes_MY_HDBR_200531 : 13
 * Tree of Life - ASG : 16
 * Tree of Life - VGP : 6
 * UCL_Mice_Lung (WG)(Novaseq) : 2
 * Understanding hematopoietic stem cell mobilization and engraftment : 16
 * Using genetics to identify cell types and mechanisms underlying susceptibility to primary sclerosing cholangitis - RNA : 12
 * VGP_RNASeq : 1
 * Vaccine Pertubation Study pilot experiments : 3068
 * Vento_Placental Cell Atlas : 7
 * Vertebrate Genome Project_FISH : 13
 * WC10290_Antimicrobial resistance of Escherichia coli isolates from stool, invasive disease and environmental swabs among vulnerable populations in Kenya : 6
 * WC10314_RNA_iPS derived intestinal organoids, control untreated or treated inside or outside with CTX 48hr : 4
 * WC10479_PWGS_Group B Streptococcus from maternal carriage in Netherlands : 2
 * WGS_11pcw_fetus_hdbr_15951_DNA : 3
 * WGS_Somatic mutation in Primary Sjogren’s Syndrome : 9

Latest date in row
 * 2022-06 : 3037
 * 2022-07 : 705
 * 2022-08 : 87
 * 2022-09 : 107
 * 2022-10 : 562
 * 2022-11 : 263
 * 2022-12 : 1884
 * 2023-01 : 564
 * 2023-02 : 67
 * 2023-03 : 26
 * 2023-04 : 7
 * 2023-05 : 3
 * 2023-06 : 33
 * 2023-07 : 1
 * 2023-08 : 36
 * 2023-09 : 16
 * 2023-10 : 12
 * 2023-11 : 2
 * 2023-12 : 1
 * 2024-01 : 308
 * 2024-02 : 17
 * 2024-03 : 171
 * 2024-04 : 207
 * 2024-05 : 79
 * 2024-06 : 708

Barcode prefix
 * CGAP- : 192
 * DN : 1669
 * FB : 1
 * FD : 59
 * FF : 147
 * FR : 27
 * FS : 1
 * ND : 9
 * NR : 5
 * NT : 6786
 * PLTE- : 2
 * SQPP- : 5
